src/server/app.ts

```diff
@@ -141,17 +141,17 @@ export default function createApp(serverSettings: ServerSettings, settingsManage
   attachRouter("/", express.static(join(__dirname, "../../build/public")));
   attachRouter("/", express.static(join(__dirname, "../../assets")));
 
-  attachRouter(serverSettings.readinessEndpoint, readinessRouter(settingsManager.sourcesGetter));
```

Comment:
Generally, I try to pass just setting manager to all routers. It should be router responsibility do declare only needed subset of `SettingsManager`.